encord/dataset.py

```diff
@@ -311,6 +312,29 @@ def run_ocr(self, image_group_id: str) -> List[ImageGroupOCR]:
         """
         return self._client.run_ocr(image_group_id)
 
+    def get_bulk_data(
```

Comment:
I'm unsure about the naming, we can debate it in a bit but I tend to think that `get_data_infos` might be a better name actually. 

Also, I think we should write a DEPRECATED warning for the other `get_data` function, given that it is just incorrect at this point and unusable.